index.js

```diff
@@ -1 +1,25 @@
-// code your solution here
+function saturdayFun(activity = "roller-skate") {
+  switch (activity) {
+    case "bathe my dog":
+      activity = "bathe my dog";
+  }
+  return `This Saturday, I want to ${activity}!`;
+}
```

Comment:
I think this would also pass the test: 
```suggestion
function saturdayFun(activity = "roller-skate") {
  return `This Saturday, I want to ${activity}!`;
}
```